dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -462,15 +462,15 @@ ANNO_CLASSSOURCE_FILE_SCAN_EXCEPTION.useraction=See the exception and the associ
 # # {4} The error message from the exception
 ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION=CWWKC0070W: [ {0} ]: The close of resource [{1}] under root [{2}] for class [{3}] failed. Error message: {4}
 ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.explanation=The close of the file caused an exception.
-ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_FILE_CLOSE_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.